File: CourseHomeworkNine/app/src/main/java/com/example/coursehomeworknine/SunObject.java
package com.example.coursehomeworknine;

import com.google.gson.annotations.SerializedName;

public class SunObject {

    @SerializedName("sunrise")
    private Long sunrise;
    @SerializedName("sunset")
    private Long sunset;

    public Long getSunset() {
        return sunset;
    }

    public void setSunset(Long sunset) {
        this.sunset = sunset;
    }

    public Long getSunrise() {
        return sunrise;
    }

    public void setSunrise(Long sunrise) {
        this.sunrise = sunrise;
    }
}

File: CourseHomeworkTen/app/src/main/java/com/example/coursehomeworkten/SunObject.kt
package com.example.coursehomeworkten

import com.google.gson.annotations.SerializedName




class SunObject {
    @SerializedName("sunrise")
    private var sunrise: Long? = null

    @SerializedName("sunset")
    private var sunset: Long? = null

    fun getSunset(): Long? {
        return sunset
    }

    fun setSunset(sunset: Long?) {
        this.sunset = sunset
    }

    fun getSunrise(): Long? {
        return sunrise
    }

    fun setSunrise(sunrise: Long?) {
        this.sunrise = sunrise
    }
}